api/descriptions/authenticate/authenticate.md

```diff
@@ -1,3 +1,3 @@
-Retrieve an access token using your credentials.
-Valid tokens are required to access the rest of the Twistlock API.
-The Twistlock API can also be accessed using [basic auth](https://docs.twistlock.com/docs/latest/api/access_api.html).
+Retrieves an access token using your credentials. Valid tokens are required to access the rest of the Twistlock API.
```

Comment:
@KICKdesigns Two things:

1. We've got an internal style of one sentence per line. Please maintain this style for us.

2. Please rename all `Twistlock` strings to either `Prisma Cloud` or `Prisma Cloud Compute`. 

Background: Twistlock was acquired by Palo Alto Networks, and incorporated into the Prisma Cloud product line.

The difference is that Prisma Cloud refers to our entire suite of cloud security products, of which Twistlock is a part. Prisma Cloud Compute refers specifically to the Twistlock functionality. I realize that the three word "Prisma Cloud Compute" can be tedious at times, so feel free to use either at your discretion.

Reply:
@iansk no problem. Will apply this across the board. For the one sentence per line - do you want it to be a regular line break or do you want a space between each line of text?

Reply:
@KICKdesigns Just a regular line break. For example, this would be one para:

```
This is a sentence.
This is another sentence.
```

If you need a second para, then add a second line break after the last line.